client/src/components/video.vue

```diff
@@ -678,6 +681,46 @@
         }, 100)
       }
     }
+    onTouchHandler(event: TouchEvent) {
+      let touches = event.changedTouches
+      let first = touches[0]
+      let type = ''
+      switch (event.type) {
+        case 'touchstart':
+          type = 'mousedown'
+          break
+        case 'touchmove':
+          type = 'mousemove'
+          break
+        case 'touchend':
+          type = 'mouseup'
+          break
+        default:
+          return
+      }
+
+      const simulatedEvent = document.createEvent('MouseEvent')
+      simulatedEvent.initMouseEvent(
+        type,
+        true,
+        true,
+        window,
+        1,
+        first.screenX,
+        first.screenY,
+        first.clientX,
+        first.clientY,
+        false,
+        false,
+        false,
+        false,
+        0 /*left*/,
+        null,
+      )
+
+      first.target.dispatchEvent(simulatedEvent)
+      event.preventDefault()
```

Comment:
We already have `.stop.prevent` in the vuejs event, so this line can be removed.